tests/PHPUnit/Unit/DataTable/Filter/SortTest.php

```diff
@@ -61,7 +61,7 @@ public function testMissingColumnValuesShouldAppearLastAfterSortAsc()
                                       )));
         $filter = new Sort($table, 'count', 'asc');
         $filter->filter($table);
-        $expectedOrder = array('nintendo', 'ask', 'amazing', 'nocolumnbis', 'nocolumn', 'summary');
+        $expectedOrder = array('nintendo', 'ask', 'nocolumnbis', 'nocolumn', 'amazing', 'summary');
```

Comment:
Now we do actually have a secondary sort column when both values are not set, this was not the case before. Before the order was random.